Docs: Read the Docs for Science (#9447)
readthedocs/readthedocs.org: docs/user/science.rst

Read the Docs for Science
=========================

> Documentation and technical writing are broad fields that include many science activities - building publications, academic writing, educational resources and interactive data science and data journalism.

Reviewer: ```suggestion
Documentation and technical writing are broad fields that include many scientific activities. 
This includes building publications, academic writing, educational resources, interactive data science, and data journalism.
```

> Let's explore the overlap between features needed for software documentation and for instance academic writing. Here's a brief overview of some of the features that people in science and academic writing love about the many tools used on Read the Docs:

Reviewer: ```suggestion
Let's explore the overlap of features for software documentation and academic writing.
Here's a brief overview of some features that people in science and academic writing love about Read the Docs:
```

🪄 Easy to use
  You don't have to be a programmer. Use familiar Markdown-derived syntax and write your formulas with LaTeX.
🔣 Formulas and proofs
  Use symbols familiar from math and physics, build advanced proofs.
🧮 Results directly imported

>   Graphs, tables etc. are computed when the latest version of your website is built. Your results can be directly imported in their latest versions.

Reviewer: What is directly imported? I feel like there's better wording here, but I don't know exactly what it's trying to convey.

Perhaps:

```suggestion
  Graphs, tables etc. are computed when the latest version of your website is built. All code examples on your website are validated each time you build.
```

📐 Widgets, widgets and more widgets

>   Design your science project's layout and components with widgets from a rich eco-system of Open Source extensions built for many purposes.

Reviewer: What are widgets? Example here might help.

Author: Good point! This will be elaborated in the new PR

📊 Maps, graphs, plots and more

>   Present results with plots, graphs, images and let users interact directly with your datasets and algorithms.

Reviewer: Is this the same as Formulas and proofs? I feel like we already have too many bullets without any backing evidence. We definitely needs links and examples here.

Author: I wanted to shape the content like this: 1) Create a compelling set of arguments for the "generic" science user (like a lead paragraph + an easy and convincing list of features and 2) add inline links to all the significant words + 1-2 screenshots

Author: These are all merged now.

📚 Bibliographies and external links
  Maintain bibliography databases directly as code and have external links automatically verified.
📜 Modern themes and classic PDF outputs
  Use the latest state-of-the-art themes for web and have PDFs and e-book formats automatically generated.
⚙️ Automatic builds
  Build and publish your project for every addition in Git. Preview changes via Pull Requests. Receive notifications when something is wrong.
💬 Collaboration and community
  Easy access for readers to suggest changes via GitHub, GitLab etc.
🔎 Search Engine Optimization
  Don't reinvent SEO: Have SEO handled jointly by the combined best-practices from Sphinx, its themes and Read the Docs hosting.
🌱 Grow your own solutions
  The eco-system is Open Source and makes it accessible for anyone with Python skills to build their own extensions.

We want science communities to use Read the Docs and to be part of the documentation community 💞

Getting started: Jupyter Book
-----------------------------

.. Left this out:

> .. Jupyter Book is a popular and well-supported platform for Read the Docs, and we aren't shy to call it the no. 1 documentation tool for science.

Reviewer: I think we can say it's well supported without making claims we can't back up.

Author: Yes, I left it as a comment, I'm not sure how much "marketing speak" is appropriate for our docs, since the whole format is "documentation", so a level of objective language is to be expected + the readers are from science/academia :)

..
.. Creating and sharing *computational documents* has long been the goal of Jupyter Notebook and JupyterLab. They offer a simple, streamlined and document-centric experience.
..
.. TODO: Insert a screenshot with a caption linking to the Jupyter Book gallery?

> :doc:`Jupyter Book <jupyterbook:intro>` on Read the Docs brings you the rich experience of computated `Jupyter <https://jupyter.org/>`__ documents built together with a modern documentation tool. The results are beautiful and automatically deployed websites, built with Sphinx and :doc:`Executable Book <executablebook:index>` + all the extensions available in this ecosystem. You even get PDF and eBook versions and since the code behind is managed via Git, all your readers can help you build it as (if you want).

Reviewer: > all your readers can help you build it as (if you want)

This feels incomplete.

Author: It is, and it is also gone now, since it pertains more to the feature list.
